feature-group-definitions/js-modules-for-workers.dist.yml

```diff
@@ -0,0 +1,20 @@
+# Generated from: js-modules-for-workers.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Worker JavaScript modules
+description: 'The `Worker()` constructor accepts `{ type: "module" }` to load scripts that use `import` and `export`. Also known as ECMAScript modules or ESM in workers.'
+spec:
+  - https://html.spec.whatwg.org/multipage/workers.html#dom-worker-dev
+  - https://html.spec.whatwg.org/multipage/workers.html#shared-workers-and-the-sharedworker-interface:dom-sharedworker-2
+status:
+  baseline: false
+  support:
+    chrome: "80"
+    edge: "80"
+    firefox: "114"
+    firefox_android: "114"
+    safari: "15"
+    safari_ios: "15"
+compat_features:
+  - api.Worker.Worker.ecmascript_modules
+  - api.SharedWorker.SharedWorker.ecmascript_modules
```

Comment:
It wasn't 100% clear to me whether to include `SharedWorker` in this. Without it, this feature is Baseline low (Chrome for Android does not support shared workers.)

Reply:
Since it does affect the status, I say let's split it. We'll have these 3 "subfeatures" of JS modules:

- JavaScript modules in dedicated workers
- JavaScript modules in service workers
- JavaScript modules in shared workers